contracts/0.4.24/oracle/LidoOracle.sol

```diff
@@ -414,6 +421,16 @@ contract LidoOracle is ILidoOracle, AragonApp {
         POST_COMPLETED_TOTAL_POOLED_ETHER_POSITION.setStorageUint256(ISTETH(lido).totalSupply());
         PREV_COMPLETED_EPOCH_ID_POSITION.setStorageUint256(LAST_COMPLETED_EPOCH_ID_POSITION.getStorageUint256());
         LAST_COMPLETED_EPOCH_ID_POSITION.setStorageUint256(epochId);
+
+        // call the quorum delegate
+        uint256 quorumDelegateRaw = QUORUM_DELEGATE_POSITION.getStorageUint256();
+        (uint256 lastTotalPooledEther,
+         uint256 prevTotalPooledEther,
+         uint256 timeElapsed) = getLastCompletedReports();
+        address(quorumDelegateRaw).call(bytes4(quorumDelegateRaw >> 160),
```

Comment:
if delegate is not 0

Reply:
it can be 0

Reply:
it is OK: in this case the call simply returns `false` (which we ignore anyway)

Reply:
moved to using interface `IQuorumCallback` and thus added the check.

Reply:
I mean that callback can be simply not set. If you call `0x0.processLidoOracleReport(...)` we'll revert.

Reply:
Yes, now did so. And tested.
I meant that previously when I did `address(callback).call(bytes4(keccak256("...")), ...)`, this construction would not revert even if the `callback` was 0.

Reply:
huh, right. I assumed calling an EOA would revert.